src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsTag.cs

```diff
@@ -57,13 +57,20 @@ public override GraphicsResult GetGraphics(IWpfTextView view, Geometry bounds, T
             };
 
             var idRun = GetRunForId(out var hyperlink);
-
+            var editorBackground = (Color)_editorFormatMap.GetProperties("TextView Background")["BackgroundColor"];
             if (hyperlink is null)
             {
                 block.Inlines.Add(idRun);
             }
             else
             {
+                // If we are in dark mode, then we want to change the color of the link to be lighter than the
+                // default color to meet accessibility standards.
+                if (editorBackground == Color.FromArgb(255, 30, 30, 30))
```

Comment:
this feels hardcodey and wrong, please let me know if there is a better way to do this

Reply:
does not look like this anbymore